README.rst

```diff
@@ -18,9 +26,104 @@ There is no released code API.
   Sometimes, it may not work at all.
 
 Each official release is tagged in the repository. The `change
-history`_ describes every version.
+history` _describes every version.
+
+ROS usage
+---------
```

Comment:
It's great to have this documentation, but I'll be honest that READMEs are about my leave favourite place to put it. I'm wondering if it's worth considering documenting this package using Sphinx, and then having the README and package wiki page direct users there? It seems especially appropriate as you are explicitly exposing a Python API. See for example: http://docs.ros.org/indigo/api/capabilities/html/